ts/utils/web3_connector.ts

```diff
@@ -0,0 +1,224 @@
+import { getContractAddressesForNetworkOrThrow } from '@0x/contract-addresses';
```

Comment:
This was mostly copied from the Governance [VoteForm](https://github.com/0xProject/website/blob/development/ts/pages/governance/vote_form.tsx) component and refactored into an isolated Web3 utility.